src/routes/resource-edit.js

```diff
@@ -105,7 +105,7 @@ class ResourceEditRoute extends Component {
       });
       model.isSelected = values.isSelected;
       model.url = customUrl;
-      model.visibilityData.isHidden = isVisible === 'false';
+      model.visibilityData.isHidden = !isVisible;
```

Comment:
In this file, there is a  mutation of the `modal` prop. It's not a React way of changing data. I know, Jeffrey, it's not your changes, but maybe it's a point to be discussed and reworked.

Reply:
Commit added that improves that. ✨